subprojects/libft/libft.h

```diff
@@ -21,13 +21,6 @@
 ** TYPES
 */
 
-typedef struct	s_list
-{
-	void			*content;
-	size_t			content_size;
-	struct s_list	*next;
-}				t_list;
```

Comment:
Would t_list be worth preserving as a system for strings? using t_list_conn for actual linked list functionality

Reply:
How would the `t_list` be used for strings?

Reply:
that's simply my most common use-case for t_lists, to hold strings in a list, sometimes even on their own, as i find the len-char * pairing useful

Reply:
Hmm, I don't like the idea of having two conflicting list types in the `libft`. We could add a `t_string` to our project, but I'm hesitant to add another type to libft right away. Maybe we could have such a type in our project for a while before finalizing it as a `libft` type?

Reply:
how would they be conflicting types?